submissions/chimoney-telegram-bot/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Could you please add a readme to your bot folder with instructions on how to start and use this code?